tv/tv.cpp

```diff
@@ -177,6 +179,12 @@ llvm::cl::opt<int> max_subprocesses(
                    "(default=128)"),
     llvm::cl::cat(TVOptions), llvm::cl::init(128));
 
+llvm::cl::opt<long> subprocess_timeout(
+    "tv-subprocess-timeout",
+    llvm::cl::desc("Maximum time, in milliseconds, that a parallel TV call "
```

Comment:
I would suggest we switch this to seconds. This timeout is very macroscopic; it's unlikely we want to go down to ms.

Reply:
can do. only reason I used milliseconds here was for consistence with our other timeouts.

Reply:
I would rather go with seconds, otherwise we'll be counting bazillion zeros on the command line.

Reply:
done, and this is the only granularity supported by alarm() anyway :)